src/store/index.js

```diff
@@ -1,17 +1,16 @@
-import { createStore, applyMiddleware } from 'redux';
+import { createStore, compose, applyMiddleware } from 'redux';
 import thunk from 'redux-thunk';
 import { createEpicMiddleware } from 'redux-observable'; // TODO: Stop using
 import reducer from '../reducers';
 import rootEpic from '../epics'; // TODO: Stop using
 
 const epicMiddleware = createEpicMiddleware(rootEpic);
 
+const composeEnhancer = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+
 const store = createStore(
     reducer,
-    process.env.NODE_ENV === 'development' &&
-        window.__REDUX_DEVTOOLS_EXTENSION__ &&
-        window.__REDUX_DEVTOOLS_EXTENSION__(),
-    applyMiddleware(thunk, epicMiddleware)
+    composeEnhancer(applyMiddleware(thunk, epicMiddleware))
```

Comment:
Is this functionally the same?

Reply:
Not sure - the previous setup gave an error on the store. New syntax is from here:
https://redux-observable.js.org/docs/basics/SettingUpTheMiddleware.html#redux-devtools

Reply:
Added a check if in development mode.